### ISSUE TITLE
Accessibility: Audit Storefront theme using AXE Browser extension.

### ISSUE THREAD

**Issue by nerrad** [woocommerce/storefront#1247]
This should be focused on an audit of the storefront theme using [AXE browser extensions](https://www.deque.com/axe/) to surface obvious accessibility issues.  Any a11y problems discovered should have an issue created in the Storefront repo identifying what the problem is and the error reported.

The audit should include all frontend flows with the Storefront theme.  This issue is complete once that audit is done (fixing all the exposed problems is not a requirement to complete this issue).
